### PR TITLE
Unskip tests

### PR DESCRIPTION
**Pull request by aicioara**
Unskips few tests. Created this now to unblock #2092
Part of #1921 
- Upgraded Firefox and Chrome on SauceLabs to the latest version
- Increased tolerance of FireFox pixel precision to 2

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -134,10 +134,10 @@ module.exports = function(grunt) {
 
   var browsers = [{
     browserName: "firefox",
-    version: "30"
+    platform: "linux"
   }, {
     browserName: "chrome",
-    version: "35"
+    platform: "linux"
```
Comment (bluong) [inline Gruntfile.js]: Why "linux"?  At the very least we probably want to go something like... Windows?

Comment (rcchen) [inline Gruntfile.js]: Yeah we should default to Windows for these.

Comment (aicioara) [inline Gruntfile.js]: That was the original default. I tried to minimize the changes. You want Windows?

File: test/globalInitialization.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ before(() => {
   if (window.PHANTOMJS) {
     window.Pixel_CloseTo_Requirement = 2;
   } else if (isFirefox) {
-    window.Pixel_CloseTo_Requirement = 1;
+    window.Pixel_CloseTo_Requirement = 2;
```
Comment (bluong) [inline test/globalInitialization.ts]: Why this change?

Comment (rcchen) [inline test/globalInitialization.ts]: What's the reason for this? (why is 2 a more correct number than 1)

File: Gruntfile.js (hunk 1 above)
Comment (bluong) [inline Gruntfile.js]: Most of our users will probably be on Windows.  We probably want to align to that fact.

File: test/globalInitialization.ts (hunk 2 above)
Comment (aicioara) [inline test/globalInitialization.ts]: The issue is that the `SVGTypeWriter` uses `getBBox()` to figure out the width and height of a text element, while our tests use `getBoundingClientRect()` for the same purpose. There is no difference between the 2 values in IE9 and Chrome, but it differs in Mozilla by 2 pixels. This was the easiest change

Comment (bluong) [inline test/globalInitialization.ts]: As long as we think we have the right number here I think it makes sense

File: Gruntfile.js (hunk 1 above)
Comment (aicioara) [inline Gruntfile.js]: Did the change and there are more tests failing, due to pixel rounding differences (I believe)... I'd file another issue for this.

File: test/globalInitialization.ts (hunk 2 above)
Comment (aicioara) [inline test/globalInitialization.ts]: We probably don't since that should be a ratio (be dependent on the size), while we use it as a constant

File: Gruntfile.js (hunk 1 above)
Comment (aicioara) [inline Gruntfile.js]: Opened this https://github.com/palantir/plottable/issues/2122

File: test/globalInitialization.ts (hunk 2 above)
Comment (aicioara) [inline test/globalInitialization.ts]: + that thing is a hack

Comment (rcchen) [inline test/globalInitialization.ts]: Can you add the explanation that you had about `getBBox()` and `getBoundingClientRect()` to the comments on the source here and annotate it with `HACKHACK:`? That way this explanation won't be lost for the future.

File: Gruntfile.js (hunk 1 above)
Comment (bluong) [inline Gruntfile.js]: Why can't we switch to Windows and then just fix the tests in accordance in this PR.  I don't mark this currently as an upgrade
